design.html

```diff
@@ -0,0 +1,1145 @@
+          <figure class="" >
+            <img class="st-row-imgs ml-lg-0 st-row-imgs-md st-row-imgslg " src="./assets/things/rankin.PNG" alt="politics">
+            <figcaption class=" pb-3 ml-lg-0 st-row-imgs st-row-imgs-md st-row-imgslg  text-justify border border-top-0" ><a class=" st-row-text font-weight-bold text-body" href="">Biden, the Moderate, Plans the Most Radical Economic Overhaul Since FDR</a><br>
+                To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to
+                strengthen the social safety net and revive prosperity. The big question: Is he up to the
+                challenge?</figcaption>
+        </figure>
+        <figure class="" >
+          <img class="st-row-imgs st-row-imgs-md ml-lg-0 st-row-imgslg" src="./assets/things/row3-2.PNG" alt="politics">
+          <figcaption class=" pb-3 st-row-imgs ml-lg-0 st-row-imgs-md st-row-imgslg text-justify border border-top-0" ><a class=" st-row-text font-weight-bold text-body" href="">Biden, the Moderate, Plans the Most Radical Economic Overhaul Since FDR</a><br>
+              To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to
+              strengthen the social safety net and revive prosperity. The big question: Is he up to the
+              challenge?</figcaption>
+      </figure>   
+         </div>
+                </div> 
+                </div>
+</section>       
+        </div>
+        </div>
+  </main> <!---->
+     
+ <section class="middle-tittle justify-content-center border-bottom pt-5"> <h3>twm</h3>
+         <div class="text-center" >
+           <span class="font-weight-bold"><img src="./assets/things/N-sign.PNG" alt="nsign">IN THE MAGAZINE</span>
+         </div>
+       </section>
+       <section class="container-fluid  "> <h3>twm</h3>
+        <div  class=" d-lg-flex  flex-lg-row d-md-flex  flex-md-row middle-stn-1" >
+          <figure>
+              <img class="middle-imgs-main mt-2 pl-2 justify-content-center w-100 " src="./assets/things/cover-june-12-2020.jpg" alt="politics">
+              <figcaption  class=" h5 middle-txtmain text-black  font-weight-bold" > 7 Wild Stories From the Singular Life of Joel Schumacher</figcaption>
+          </figure>
+          <figure>
+              <img class=" middle-imgs mt-4 ml-3 w-100 " src="./assets/things/bottom2.PNG" alt="politics">
+              <figcaption  class="middle-txt h5  text-justify-center ml-4 text-black font-weight-bold" > 7 Wild Stories From the Singular Life of Joel Schumacher
+                      FDR</figcaption>
+          </figure>
+          <figure>
+              <img class="  middle-imgs mt-4 ml-3 w-100" src="./assets/things/bottom1.PNG" alt="politics">
+              <figcaption  class="middle-txt h5 text-justify-center text-black font-weight-bold" >7 Wild Stories From the Singular Life of Joel Schumacher
+                      FDR</figcaption>
+          </figure>
```

Comment:
- [ ] The images still have a low resolution. Did you take print screens because of the 'Cover-World' box at the bottom? You should not do that. Use the original image and generate this item using divs and text inside with backgrounds with a position relative / absolute. Hit me on slack if you need help.

![image](https://user-images.githubusercontent.com/61479648/88105013-a3559e00-cb79-11ea-8477-47cb4e9a3407.png)

(this cover-world in the original is a text)